sechub-cli/src/daimler.com/sechub/util/zip.go

```diff
@@ -98,20 +97,30 @@ func zipOneFolderRecursively(zipWriter *zip.Writer, folder string, zContext *zip
 		if err != nil {
 			return err
 		}
-		if zContext.filename == filePath {
+		if zContext.filename == file {
 			return errors.New(TargetZipFileLoop)
 		}
 
-		fileAbs, err := filepath.Abs(filePath)
+		fileAbs, err := filepath.Abs(file)
 		if err != nil {
 			return err
 		}
 		pwdAbs, _ := filepath.Abs(".")
 		// Make zip path relative to current working directory (the usual case)
-		zipPath := strings.TrimPrefix(fileAbs, pwdAbs+"/")
-		// If we still have a / in front: use path from folder var stripped from "./" and "../"
+		zipPath := strings.TrimPrefix(fileAbs, pwdAbs)
+
+		// tribute to Windows...
```

Comment:
Please change this comment to make it more informative.

Reply:
changed